src/irmin/type/type.mli

```diff
@@ -310,6 +313,10 @@ val to_string : 'a t -> 'a -> string
 val of_string : 'a t -> 'a of_string
 (** [of_string t] parses values of type [t]. *)
 
+val boxed : 'a t -> 'a t
+(** [boxed t] is the same as [t] but with a binary representation which is
+    always boxed (e.g. top-level values won't be unboxed). *)
```

Comment:
Potentially worth explaining why this might be useful to a user & referencing `{!Unboxed}`.